test-requirements.txt

```diff
@@ -1,9 +1,11 @@
-Pytest-cov==2.2.1
-flake8==2.5.2
-coveralls==1.1
-autopep8==1.2.1
-lettuce==0.2.21
+# === Test ===
+lettuce~=0.2.21
 
 # Optional speed-up for Lettuce, but Docker image build issue needs fixing, see:
 # https://github.com/ensime/ensime-docker/pull/8
 # python-Levenshtein~=0.12
+
+# === Dev Tooling ===
+flake8
```

Comment:
I just thought releases of these are probably unlikely to break things and it's not for production code, so it's nice to get updates occasionally without thinking about it. But I can restore them if you think it's best, if used in CI some constraint is a good idea. The `~=` "compatible" operator on the minor version could be a good compromise (basically expect Semantic Versioning).